tests/corpus/assets/test_file.py

```diff
@@ -36,16 +37,10 @@ def test_duration(self):
         assert self.stereo_file.duration == pytest.approx(0.000589569161)
 
     def test_read_samples(self):
-        ref = np.array([6.10351562e-05, 9.15527344e-05, 1.22070312e-04,
-                        1.52587891e-04, 1.03759766e-03, 1.83105469e-04,
-                        2.13623047e-04, 9.15527344e-05, 1.52587891e-04], dtype=np.float32)
+        expected, __ = librosa.core.load(MONO_16K_16BIT_9, sr=None)
+        actual = self.mono_file.read_samples()
+        assert np.array_equal(actual, expected)
 
-        assert np.array_equal(ref, self.mono_file.read_samples())
-
-        ref = np.array([7.62939453e-05, -3.05175781e-05, -6.10351562e-05,
-                        3.05175781e-05, -9.15527344e-05, 3.05175781e-05,
-                        1.52587891e-05, 1.83105469e-04, -1.06811523e-04,
-                        -3.05175781e-05, -1.67846680e-04, -6.10351562e-05,
-                        1.37329102e-04], dtype=np.float32)
-
-        assert np.array_equal(ref, self.stereo_file.read_samples())
+        expected, __ = librosa.core.load(STEREO_22050_32BIT_13, sr=None)
+        actual = self.stereo_file.read_samples()
+        assert np.array_equal(actual, expected)
```

Comment:
A resampling test (`sr != None`) would be nice, too.

Reply:
yup